adaptive-education-ai/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "typescript-node-api",
+  "version": "1.0.1",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "start": "node dist/index.js",
+    "test": "mocha --reporter spec --compilers ts:ts-node/register 'src/**/*.test.ts'"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "devDependencies": {
+    "@types/body-parser": "0.0.33",
+    "@types/chai": "^3.4.34",
```

Comment:
I'm always a little hesitant on auto-updating versions. Even for minor patches, I've had things break out from underneath me due to this. That said, given the timeline I can't imagine this will cause a problem so I have no preference on leaving these or locking them down. Just wanted to mention it.